vars/folioHelm.groovy

```diff
@@ -188,6 +188,11 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
     moduleConfig['extraEnvVars:'] += [name: 'LEGACY_TOKEN_TENANTS', value: '']
   }
 
+  // Switch on Eureka related configuration in Helm Charts for Modules
+  if (ns.enableEureka && moduleName.startsWith('mod-')) {
```

Comment:
@eldiiar-duishenaliev 

I have concern for recommended approach.
All common backend modules (not Eureka related) also start with "mod-" prefix.

Does it allign with our goals in this case?